### DISCUSSION TITLE
v8 missing option to scan remote repos?

### DISCUSSION THREAD

**Discussion by mickutz** [zricethezav/gitleaks#706] [General]
Hi,

Just noticed v8 was released a couple of days ago. I do not understand why there's no option to scan remote repositories though?

**Comment by zricethezav** [comment]
I dont want to maintain cloning repos anymore. Most of the issues created were problems with authentication and I didn't have the bandwidth to address all of them. I also didn't like it from a design standpoint. You can easily script it
```bash
git clone {remote}
cd {remote}
gitleaks detect
```

**Comment by mickutz** [comment]
Hi @zricethezav
Thanks for the reply, that's exactly what I ended up doing ;).
